recipes/python-logstash/meta.yaml

```diff
@@ -0,0 +1,38 @@
+{% set name = "python-logstash" %}
+{% set version = "0.4.6" %}
+
+ package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+ source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 10943e5df83f592b4d61b63ad1afff855ccc8c9467f78718f0a59809ba1fe68c
+
+ build:
+  number: 0
+  noarch: python
+  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed -vv "
+
+ requirements:
+  host:
+    - pip
+    - python
+  run:
+    - python
+
+ test:
+  imports:
+    - logstash
+
+ about:
+  home: https://github.com/vklochan/python-logstash
+  license: MIT
+  license_family: MIT
+  license_file: LICENSE
+  summary: Python logging handler for Logstash.
+
+ extra:
+  recipe-maintainers:
+    - pg-hbk
```

Comment:
@pg-hbk are you ok with being a maintainer?

Reply:
Yes